### DISCUSSION TITLE
@enumerated cause bytes must not be nullable error without null safety

### DISCUSSION THREAD

**Discussion by ductranit** [isar/isar#645] [Help]
Hi there,
Here is my code
```
enum BookType { Type1, Type2 }
class Book {
  @enumerated
   BookType type = BookType.Type1;
}
```

I got error when running `flutter pub run build_runner build`
```
[SEVERE] isar_generator:isar_generator on lib/models/book.dart (cached):

Bytes must not be nullable.
package:books/models/book.dart:14:12
   ╷
14 │   BookType type = BookType.Type1;
   │            ^^^^^^^^
   ╵
``` 
It only happens on my current project because I'm not using null safety (I can't migrate null safety atm). It works fine on new project with null safety.
I wonder is there any way to bypass this check?

**Comment by simc** [comment]
No I'm very sorry! The generator assumes null-safety is enabled. It would be an incredible amount of work do maintain both versions.
